src/ChartVis.js

```diff
@@ -2,66 +2,62 @@ import React, { Component } from 'react'
 // import Buttons from './Buttons'
 // import YearRange from './YearRange'
 
+/* global fetch */
 
 const SERVER_ROOT = 'http://api.worldbank.org/'
 const USER_PATH1 = `${SERVER_ROOT}countries/it/indicators//NY.GDP.MKTP.CD?format=json&per_page=500&date=1995:2015`
 
-
 export default class ChartVis extends Component {
-    // 30.11.2017. Restart here. 
-    constructor(props) {
-        super(props)
-
-        this.state = {
-            data: []
-        }
-    }
-
-    componentWillMount () {
-        fetch(`${USER_PATH1}`)
-            .then((response) => response.json())
-            .then((data) => {
-                const filteredData = data[1].map(function(datapoint,i) { 
-                    return {
-                        'id': datapoint.country.id,
-                        'year': datapoint.date,
-                        'value': datapoint.value 
-                    }
-                })
-                this.setState({ data: filteredData }) // TODO: Check the JSON response object. 
-            })
-            .catch(error => console.log(error))
-    }
-
-    componentDidMount() {
-        // This will call the d3 to show the chart 
-    }
-
-    d3_function() {
-    }
-
-    
-    Italy_GDP_Chart(data) {    
-    }
-
-    render() {
+  // 30.11.2017. Restart here.
+  constructor (props) {
+    super(props)
 
-        // call js functions here 
-        console.log(this.state.data)
-        return (
-            // chart id is for SVG 
-            <section className="all_container">
-
-                <div className='container'>
-                    <div id='chart'></div>
-                    <div id='slider'></div>
-                    <p id='range-label'></p>
-                </div>
-
-            </section>
-
-        )
+    this.state = {
+      data: []
     }
+  }
+
+  componentWillMount () {
+    fetch(`${USER_PATH1}`)
+      .then((response) => response.json())
+      .then((data) => {
+        const filteredData = data[1].map(function (datapoint, i) {
+          return {
+            'id': datapoint.country.id,
+            'year': datapoint.date,
+            'value': datapoint.value
+          }
+        })
+        this.setState({ data: filteredData }) // TODO: Check the JSON response object.
+      })
+      .catch(error => console.log(error))
+  }
+
+  componentDidMount () {
+    // This will call the d3 to show the chart
+  }
+
+  d3Function () {
+  }
+
+  italyGDPChart (data) {
+  }
+
+  render () {
+    // call js functions here
```

Comment:
If this were a professional situation, I would tell you to pull this comment and the one on line 50. However, if these help you get started, then go ahead and leave them in. 😄

Reply:
Yes, this helps me. But I will remember not to do this in a  professional situation.